arrow_pd_parser/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.4"
+__version__ = "1.0.5"
```

Comment:
I'd say this is a feature add, so `1.1.0`?

Reply:
Done - [c44ff2a](https://github.com/moj-analytical-services/mojap-arrow-pd-parser/pull/73/commits/c44ff2aab34ccdc79a0da089a9f5cfa78b963768)